Dockerfile.dev

```
@@ -8,10 +8,11 @@ FROM golang:1.25 AS go-builder
 WORKDIR /go/src/github.com/perses/perses
 COPY . .
 COPY --from=node-builder /app/ui/app/dist ui/app/dist
-RUN mkdir /perses
-RUN mkdir /plugins
-RUN make build-api
-RUN make build-cli
+RUN <<EOR
```

```suggestion
RUN mkdir /perses \
        && mkdir /plugins \
        && make build-ap \
        && make build-cli
```

This is the usual syntax to chain bash command.

---

Done.